integration_tests/e2e/showReferral.cy.ts

```diff
@@ -66,13 +67,15 @@ context('Viewing a submitted referral', () => {
     }),
     addedByDisplayName: StringUtils.convertToTitleCase(user.name),
   }
+  let randomRole: 'ACP_PROGRAMME_TEAM' | 'ACP_REFERRER'
+  let paths: typeof assessPaths | typeof referPaths
 
   beforeEach(() => {
+    randomRole = Math.random() < 0.5 ? 'ACP_PROGRAMME_TEAM' : 'ACP_REFERRER'
```

Comment:
Is this preferable to using `faker.helpers.arrayElement(['ACP_PROGRAMME_TEAM', 'ACP_REFERRER'])` like we do elsewhere?

Reply:
Ha, that would've been smarter. I think I'm just not used to thinking in Faker on the integration test side!